assignments/arrays.js

```diff
@@ -63,32 +63,52 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a `+ (inventory[33].car_year)+`, `+ (inventory[33].car_make)+" "+ (inventory[33].car_model));
```

Comment:
Remember that array indices start at `[0]`. The actual output here is the information for the car with the ID `34`. This is commonly known as an off-by-one error - make sure to always count from 0 for arrays to avoid it.

Right now, this is still a small mix-up, but it can take up some time for debugging especially if it happens in a critical area of a larger project.